src/main/java/meteordevelopment/meteorclient/mixin/InGameHudMixin.java

```diff
@@ -122,4 +123,13 @@ private int onRenderStatusBars(int y) {
     private LivingEntity modifyGetHeartCount(LivingEntity entity) {
         return Modules.get().get(NoRender.class).noMountHiding() ? null : entity;
     }
+
+    @ModifyVariable(method="addChatMessage",at=@At("HEAD"),index = 2, argsOnly = true)
+    Text real(Text value) {
+        /*String test = "\u002a\u002fString search = "testtest123";
+        String a = value.getString();
+        while(a.contains(search)) a.replace(search, "<censored>");
+        /**/
+        return value;
+    }
```

Comment:
uh this should be removed probably

Reply:
> uh this should be removed probably

tried to remove it but removing it doesnt let the client start anymore and i do not know why. regardless, its commented out

Reply:
> doesnt let the client start anymore

how <img src="https://cdn.discordapp.com/emojis/926476459395154000.webp?size=64" />

Reply:
> > doesnt let the client start anymore
> 
> how <img alt="skol" width="24" src="https://camo.githubusercontent.com/6b5416eb4f01ddf598763ce6431aeb6caaf9dbfc00d197f738b3dac3d277f259/68747470733a2f2f63646e2e646973636f72646170702e636f6d2f656d6f6a69732f3932363437363435393339353135343030302e776562703f73697a653d3634">

i have no idea tbh might be a mixin quirk but it always just crashes when i load a world without that in

Reply:
You literally added the mixin, how can it crash without it.

Reply:
> You literally added the mixin, how can it crash without it.

as i said it just didnt work when i removed it do not ask me why

Reply:
Try removing it again and if it doesn't work send crash report.

Reply:
if it works then I don't see the problem tbh

Reply:
> if it works then I don't see the problem tbh

agreed

Reply:
lets just get the pr over with my email is getting spammed

Reply:
> Try removing it again and if it doesn't work send crash report.
> 
> 
> 
> Your whole point with the PR is to clean the code, isn't adding in a mixin that's empty a bit counterproductive?

I guess so, but I do think this is a mixin quirk. When i remove the method, it crashes almost instantly with an "exception access violation", which means this is low level stuff in a library of java. I have no idea how it got to that but apparently i did something wrong, so i just kept it in. As people said, if it works its not a problem i guess

Reply:
Looks like you can't follow simple instruction to send the crash report  / log, guess I will have to look at it later at home.

I am not adding empty mixin.

Reply:
> Looks like you can't follow simple instruction to send the crash report  / log, guess I will have to look at it later at home.
> 
> 
> 
> I am not adding empty mixin.

there is not even a crash log, it's just a stack dump from the java jvm

Reply:
It works flawlessly without it for me.

Reply:
> Remove it

Getting cocky, hm?

Reply:
@SIMULATAN he already closed it I don't think the pr is getting merged